src/App.js

```diff
@@ -1,38 +1,17 @@
-import React, { Component } from 'react';
+import React from 'react';
 import './App.scss';
-import { MoviesList } from './components/MoviesList';
+import { Search } from './components/Search';
```

Comment:
Well, this component is actually a list of movies (not the component for searching). So this renaming is not semantical

Reply:
totally agree, thanks 😃